internal/cli/option/flag.go

```diff
@@ -147,6 +147,11 @@ func NewAlias(fs *pflag.FlagSet, stage *string, usage string) {
 	fs.StringVar(stage, NewAliasFlag, "", usage)
 }
 
+// NoHeaders adds the NoHeadersFlag to the provided flag set.
+func NoHeaders(fs *pflag.FlagSet, noHeaders *bool) {
+	fs.BoolVar(noHeaders, "no-headers", false, "Don't print headers when printing resource tables")
```

Comment:
I know that nailing this help text down was part of what you were saving for last, but I don't see anything else that needs changing... so...

My only suggestion here is tweaking the text to be a little more like this:

```
kubectl get -h | grep no-headers -A 1
    --no-headers=false:
	When using the default or custom-column output format, don't print headers (default print headers).
```

We don't have any custom-column output format (nor am I inclined to add this at the moment), so we could maye tweak this to:

```suggestion
	fs.BoolVar(noHeaders, "no-headers", false, "When using the default output format, don't print headers (default print headers).")
```

Reply:
Oh... despite not being referenced in multiple placed, I would still maybe create a constant for `"no-headers"` purely for consistency with the other flags.

Reply:
This should resolve both your comments @krancour 
https://github.com/akuity/kargo/pull/1777/commits/1df2965113b9ff0f93d0d2eb59a4510c1adfa78c